source/manual/data-gov-uk-2nd-line.html.md

```diff
@@ -0,0 +1,84 @@
+---
+owner_slack: "#govuk-platform-health"
+title: Common 2nd line support requests for data.gov.uk
+section: data.gov.uk
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2018-12-21
+review_in: 6 months
+---
+[ckan]: https://ckan.publishing.service.gov.uk
+[dgu-docs]: https://guidance.data.gov.uk
+
+This document details some of the requests that GOV.UK 2nd line support may receive regarding data.gov.uk.  [Separate documentation][dgu-docs] exists for publishers.
+
+## Users and Publishers
+Users have their own accounts and can login using either their username or email address.   A user can be a member of one or more publishing organisations, and can have either the role 'Admin' or 'Editor' for each organisation.  Users with 'Admin' status can add/remove users from the organisation for which they are an admin.  They only have 'admin' rights on their own organisation, contrasted to a 'sysadmin' who has admin rights across all organisations.
+
+### Create a user account
+There are two routes for creating an new account for a new user:
+
+1. A publishing organisation's 'Admin' user(s) can invite new users to create an account.  The 'Admin' user should follow the instructions in [Assign users to publishers](#assign-users-to-publishers-setting-user-permissions), but login as themselves rather than a sysadmin user.  This route is also described in the [publisher guidance][dgu-docs].
+2. A sysadmin user (e.g. 2nd line) can create an account for the new user.  Follow the instructions in [Assign users to publishers](#assign-users-to-publishers-setting-user-permissions) inputting the user's email address instead of their username.  An invite email is generated and sent to the publisher.
+
+> An account should only be created if the user intends to publish on data.gov.uk.  Previously users have been able to register accounts to follow publishers.  This is no longer offered.
+
+### Create a publishing organisation
+1. Login to [CKAN][ckan] as a sysadmin user (credentials are available in the `govuk-secrets` password store, under `datagovuk/ckan`).
+2. Click the 'Publishers' button in the header.
+3. Click 'Add Publisher' and complete the form.
+4. Ensure that a publishing user has been assigned as an 'Admin' for this organisation (this would normally be the person who makes the request, so they can then add further 'Editor' users without needing to contact support).
+
```

Comment:
Should we also put this warning on the "create publishing org" section since a user is being assigned admin rights there?
```suggestion

> Check the authenticity of a request before adding a user as a publisher (i.e. make sure they actually belong to the department they want to publish for, bearing in mind that some parent organisations may publish on behalf of child organisations, e.g. BEIS can publish for the Civil Nuclear Police Authority).

```